source/draw/gpu/GpuCanvas.ooc

```diff
@@ -16,50 +16,9 @@ import GpuContext, GpuImage, Mesh, GpuYuv420Semiplanar
 version(!gpuOff) {
 GpuCanvas: abstract class extends Canvas {
 	_context: GpuContext
-	_model: FloatTransform3D // Deprecated! Do not use.
-	_view := FloatTransform3D identity // Deprecated! Do not use.
-	_projection: FloatTransform3D // Deprecated! Do not use.
```

Comment:
`DrawState` contains `view`, generates `model` from `destination` and generates `projection` from `focalLength`